azure-pipelines.yml

```diff
@@ -174,6 +174,7 @@ jobs:
       (New-Object Net.WebClient).DownloadFile($downloadUrl, "compat.zip")
       Expand-Archive compat.zip -DestinationPath . -Force
       Remove-Item compat.zip
+      new-item -path "$(Build.SourcesDirectory)/compat/vcbuild" -name NoPrebuildEvent.txt -type "file" -value "Inhibit git.sln PrebuildEvent"
```

Comment:
This is incorrect.

What we need to figure out is not whether we run in a CI/PR build, but whether we can update `vcpkg` or not.

I am very much against introducing this change, as it is a clear "papering over" the actual problem: we are currently not figuring out whether `vcpkg` is initialized via cloning or, say, by copying the ready-built artifacts or not.